### PR TITLE
Create scheduled messages from scheduled messages list

### PR DESCRIPTION
**Pull request by rowanseymour**
Captura de Pantalla 2022-09-14 a la(s) 11 49 00

Captura de Pantalla 2022-09-14 a la(s) 11 48 44

### CODE REVIEW

File: temba/msgs/views.py [hunk 1]
```diff
@@ -38,75 +37,24 @@
...
 )
 from temba.utils.models import patch_queryset_count
 from temba.utils.views import BulkActionMixin, ComponentFormMixin, ContentMenuMixin, SpaMixin, StaffOnlyMixin
 
-from .models import Broadcast, ExportMessagesTask, Label, LabelCount, Media, Msg, Schedule, SystemLabel
+from .models import Broadcast, ExportMessagesTask, Label, LabelCount, Media, Msg, SystemLabel
 from .tasks import export_messages_task
 
 
-class SendMessageForm(Form):
```
Comment (rowanseymour) [inline temba/msgs/views.py:60]: moves into view class

File: temba/settings_common.py [hunk 2]
```diff
@@ -611,7 +611,6 @@
         "locations.adminboundary_boundaries",
         "locations.adminboundary_geometry",
         "msgs.broadcast.*",
-        "msgs.broadcastschedule.*",
```
Comment (rowanseymour) [inline temba/settings_common.py:614]: not a thing

File: temba/triggers/types.py [hunk 3]
```diff
@@ -26,8 +26,8 @@ class KeywordTriggerType(TriggerType):
     )
 
     class Form(BaseTriggerForm):
-        def __init__(self, user, *args, **kwargs):
-            super().__init__(user, Trigger.TYPE_KEYWORD, *args, **kwargs)
```
Comment (rowanseymour) [inline temba/triggers/types.py:30]: because I want to pass org to `ScheduleFormMixin` used by scheduled triggers

File: temba/msgs/views.py [hunk 4]
```diff
@@ -251,24 +198,125 @@ class Meta:
...
+                )
+
+        def get_queryset(self, **kwargs):
+            return super().get_queryset(**kwargs).select_related("org", "schedule")
+
+    class ScheduledCreate(OrgPermsMixin, ModalMixin, SmartFormView):
+        class Form(ScheduleFormMixin, Form):
+            omnibox = OmniboxField(
+                label=_("Recipients"),
+                required=True,
```
Comment (rowanseymour) [inline temba/msgs/views.py:227]: this doesn't appear to work hence the check in `clean` below

File: temba/msgs/views.py [hunk 5]
```diff
@@ -179,12 +128,10 @@ def get_context_data(self, **kwargs):
             dict(count=counts[SystemLabel.TYPE_ARCHIVED], label=_("Archived"), url=reverse("msgs.msg_archived")),
             dict(count=counts[SystemLabel.TYPE_OUTBOX], label=_("Outbox"), url=reverse("msgs.msg_outbox")),
             dict(count=counts[SystemLabel.TYPE_SENT], label=_("Sent"), url=reverse("msgs.msg_sent")),
+            dict(count=counts[SystemLabel.TYPE_FAILED], label=_("Failed"), url=reverse("msgs.msg_failed")),
             dict(
-                count=counts[SystemLabel.TYPE_SCHEDULED],
-                label=_("Schedules"),
-                url=reverse("msgs.broadcast_schedule_list"),
+                count=counts[SystemLabel.TYPE_SCHEDULED], label=_("Scheduled"), url=reverse("msgs.broadcast_scheduled")
```
Comment (rowanseymour) [inline temba/msgs/views.py:133]: moved to bottom like in new UI

**Review by ericnewcomer — APPROVED** [review]

**Pull request merged**